packages/modules/src/utils/Types.ts

```diff
@@ -47,31 +59,42 @@ export enum StorageType {
 }
 
 export type SessionParams = {
+  /** Redundant now as we've favoured uuid() */
   sessionID?: string;
+  /** Session Signer: viemWallet or ethers signer. Ingested when passed into smartAccount */
   sessionSigner: SupportedSigner;
+  /** The session validation module which contains the logic on how to decode the session call data. for example erc20 session validation validates the amount of erc20 transfer */
```

Comment:
session validation module is a "sub-module" smart-contract which works with session key manager validation module. It validates the userop calldata againsnt the defined session permissions (session key data) within the contract.